### DISCUSSION TITLE
Error in OpenMPI 4.0 infiniband ports while running LDT Testcases1

### DISCUSSION THREAD

**Discussion by MariRiba** [NASA-LIS/LISF#1413] [General]
Hi 

@dmocko @karsenau 

I have installed LISF_public_release_7.4.2 in my HPC Cluster and the LDT and LIS executables are successfully generated.

But when I am trying to run the testcases I am getting an error and I don't understand what the error is and how to solve it.

The command which I gave is: " ./LDT ldt.config.noah36_params"

And the error which I am getting is : 

--------------------------------------------------------------------------
By default, for Open MPI 4.0 and later, infiniband ports on a device
are not used by default.  The intent is to use UCX for these devices.
You can override this policy by setting the btl_openib_allow_ib MCA parameter
to true.

  Local host:              virgo1
  Local adapter:           hfi1_0
  Local port:              1

--------------------------------------------------------------------------
--------------------------------------------------------------------------

WARNING: There was an error initializing an OpenFabrics device.

  Local host:   virgo1

  Local device: hfi1_0

--------------------------------------------------------------------------

PLease help me

**Comment by dmocko** [comment]
Hi @MariRiba 

I really don't know how to resolve this one.  Can you ask your sys admin for assistance?
